microsoft/testsuites/display/drm_sanity.py

```diff
@@ -0,0 +1,68 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+from typing import Any, cast
+
+from assertpy import assert_that
+
+from lisa import (
+    Logger,
+    Node,
+    TestCaseMetadata,
+    TestSuite,
+    TestSuiteMetadata,
+    simple_requirement,
+)
+
+from lisa.base_tools import Cat
+from lisa.tools import Lsmod
+
+@TestSuiteMetadata(
+    area="drm",
+    category="functional",
+    description="""
+    This test suite uses to verify drm driver sanity.
+    """,
+    requirement=simple_requirement(unsupported_os=[]),
+)
+class DRM_Sanity(TestSuite):
+    @TestCaseMetadata(
+        description="""
+        This case is to check whether the drm driver registered successfully.
+
+        Step,
+        1. Check hyperv_drm driver is inserted correctly.
+        """,
+        priority=0,
+        use_new_environment=True,
+    )
+    def verify_drm_driver(self, node: Node, log: Logger) -> None:
+        lsmod = node.tools[Lsmod]
+        if lsmod.module_exists("hyperv_drm"):
+            log.info(f"hyperv_drm module present")
+            ret  = "true"
+        else:
+            log.error(f"hyperv_drm module absent")
+            ret = "false"
+        assert_that(ret).is_equal_to("true")
+
+    @TestCaseMetadata(
+        description="""
+        This case is to check whether the dri node is populated correctly.
+
+        Step,
+        1. Check dri node is correctly populated
+        """,
+        priority=0,
+    )
+    def verify_dri_node(self, node: Node, log: Logger) -> None:
+        cat = node.tools[Cat]
+
+        dri_path = "/sys/kernel/debug/dri/0/name";
+        dri_name = cat.read(dri_path, sudo=True, force_run=True)
+        assert_that(dri_name).matches("hyperv_drm")
+
+    def before_case(self, log: Logger, **kwargs: Any) -> None:
```

Comment:
It doesn't need to be copied into test cases. It's for example purpose.

Reply:
I see, will correct